src/third_party/win32/getopt.h

```diff
@@ -0,0 +1,136 @@
+/* Getopt for Microsoft C
```

Comment:
Please add a `src/third_party/win32/getopt.*` entry under _Files derived from other sources_  in `LICENSE.adoc`.